tests/pipeline/test_generate.py

```diff
@@ -265,6 +265,28 @@ def test_with_env_defaults(self, snapshot: SnapshotTest):
         enriched_pipeline: list = yaml.safe_load(result.stdout)
         snapshot.assert_match(enriched_pipeline, "test-pipeline")
 
+    def test_with_env_all_resources(self, snapshot: SnapshotTest):
```

Comment:
what is this testing? (sorry if that's a stupid question, but I didn't get it)

Reply:
_There is no stupid questions only stupid answers_ 🧘‍♂️

It is just a mock test I added to check the debug logs. The folder contains all possible files with their envs (config.yaml, config_dev.yaml, pipeline.yaml,...)
It was easier for me to visualise the pickup file logs. This would be removed and later on I will add log captors for the picked up files!